modules/web/js/ballerina/utils/fragment-utils/index.js

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
We currently don't use index.js in our code so shall we stick with a proper file name.

Reply:
I have separated `fragment-utils.js` file into a submodule. index.js is intended to expose the API. importing fragments-utils is same as before.